### DISCUSSION TITLE
Not being able to view all the labels

### DISCUSSION THREAD

**Discussion by nozwock** [community/community#51348] [Mobile] [answered]
### Select Topic Area

Bug

### Body

Can't fully scroll to view all the labels.
![Screenshot_2023-03-29-21-51-31-71_320a9a695de7cdce83ed5281148d6f19](https://user-images.githubusercontent.com/57829219/228605523-7edcc6a8-a8ca-482f-8a64-248310f5b85b.jpg)
This is the max I'm able to scroll down.

**Comment by iskae** [comment, marked as answer]
Hey @nozwock 👋 

Thanks a lot for reaching out and your bug report ❤️ 

I was able to reproduce the issue and created an internal issue for tracking purposes 👍 We will try to fix it as soon as possible 🙇 

Please stay tuned for the future versions 🙇 

Cheers!

**Reply by Drawsi** [reply]
Same problem here, it does work on web though.
